index.js

```diff
@@ -15,23 +15,26 @@ module.exports = {
     'no-console':  [`warn`, {allow: [`info`, `warn`, `error`]}],
     'no-debugger': [`warn`],
 
-    'arrow-parens':                [`error`, `as-needed`],
-    'camelcase':                   [`error`, {'properties': `always`}],
-    'comma-dangle':                [`error`, `always-multiline`],
-    'comma-spacing':               [`error`, {'before': false, 'after': true}],
-    'eol-last':                    [`error`],
-    'eqeqeq':                      [`error`],
-    'indent':                      [`error`, 2, {SwitchCase: 1}],
-    'linebreak-style':             [`error`, `unix`],
-    'multiline-ternary':           [`error`, `never`],
-    'no-trailing-spaces':          [`error`],
-    'no-unused-expressions':       [`error`],
-    'no-use-before-define':        [`error`, {classes: false}],
-    'object-curly-spacing':        [`error`, `never`],
-    'object-shorthand':            [`error`, `always`],
-    'quotes':                      [`error`, `backtick`],
-    'semi':                        [`error`, `always`],
-    'space-before-blocks':         [`error`, `always`],
-    'space-before-function-paren': [`error`, {anonymous: `never`, named: `never`, asyncArrow: `always`}],
+    'arrow-parens':                [`warn`, `as-needed`],
+    'camelcase':                   [`warn`, {'properties': `always`}],
+    'comma-dangle':                [`warn`, `always-multiline`],
+    'comma-spacing':               [`warn`, {'before': false, 'after': true}],
+    'eol-last':                    [`warn`],
+    'eqeqeq':                      [`warn`],
+    'indent':                      [`warn`, 2, {SwitchCase: 1}],
+    'key-spacing':                 ['warn', {beforeColon: false, afterColon: true, mode: `minimum`}],
+    'keyword-spacing':             [`warn`],
+    'linebreak-style':             [`warn`, `unix`],
+    'no-multi-spaces':             ['warn', {exceptions: {Property: true }}],
+    'multiline-ternary':           [`warn`, `always-multiline`],
+    'no-trailing-spaces':          [`warn`],
+    'no-unused-expressions':       [`warn`],
+    'no-use-before-define':        [`warn`, {classes: false}],
+    'object-curly-spacing':        [`warn`, `never`],
+    'object-shorthand':            [`warn`, `always`],
+    'quotes':                      [`warn`, `backtick`],
+    'semi':                        [`warn`, `always`],
+    'space-before-blocks':         [`warn`, `always`],
+    'space-before-function-paren': [`warn`, {anonymous: `never`, named: `never`, asyncArrow: `always`}],
```

Comment:
IMO this isn't the right place to define warn-only behavior. We want to preserve the capability to distinguish between warnings and errors, and we can determine build-time treatment of each with eslint-loader options rather than steamrolling the base linter config, e.g.:
```js
// dev mode
eslint: {
  failOnWarning: false,
  failOnError: false,
}
```

Reply:
reverted. I didn't know about this